components/texas-instruments/ISO1540.stanza

```diff
@@ -18,18 +18,16 @@ public pcb-component component :
   manufacturer = "Texas Instruments"
   mpn = "ISO1540DR"
   description = "2.5-kVrms isolated bidirectional clock, bidirectional I2C isolator"
+  port p: i2c[[1 2]]
   pin-properties :
     [pin:Ref | pads:Int ... | side:Dir]
-    [GND1    | 4            | Down    ]
-    [GND2    | 5            | Down    ]
-    [SCL1    | 3            | Left    ]
-    [SCL2    | 6            | Right   ]
-    [SDA1    | 2            | Left    ]
-    [SDA2    | 7            | Right   ]
-    [VCC1    | 1            | Up      ]
-    [VCC2    | 8            | Up      ]
+    [GND1     | 4            | Down    ]
+    [GND2     | 5            | Down    ]
+    [p[1].scl | 3            | Left    ]
```

Comment:
Please confirm this style with Duncan. The code works, but Duncan may want to have the `pcb-component` definition mirror the datasheet as much as possible. You can then move the use of the proper bundle into the `pcb-module` definition.

Reply:
I reviewed this and like it. Actual ports on components are very handy.

Reply:
Perfect. Thanks for confirming.